### PR TITLE
Install command

### PR DESCRIPTION
**Pull request by menasrac**
### Description

This pull request is to precise the specific command to install Radis depending on the plateform.

### CODE REVIEW

File: radis/lbl/factory.py [hunk 1]
```diff
@@ -317,6 +317,44 @@ class SpectrumFactory(BandFactory):
...
+    - 'transmittance': plot transmittance vs. wavenumber
+    - 'radiance': plot radiance vs. wavenumber
+    - 'intensity': plot spectral intensity vs. wavenumber
+    - 'lines': plot individual spectral lines
+    
+    Each plot type also has additional optional parameters that can be passed to `plot_options`.
+    
+    Parameters
+    ----------
+    kind : str, optional
```
Comment (minouHub) [inline radis/lbl/factory.py:335]: we usually employ the keyword 'var' for variable

File: README.rst [hunk 2]
```diff
@@ -33,6 +33,20 @@ or with *mamba*::
 
     mamba install radis
 
+
+**Remark**
+Depending on your platform you should write
```
Comment (minouHub) [inline README.rst:38]: you *may* write. There are so many options. For dev, 'pip install -e .' is useful to also install dependencies

**Comment by minouHub** [comment]
added a few comments. Good to go otherwise

File: README.rst [hunk 3]
```diff
@@ -33,6 +33,20 @@ or with *mamba*::
 
     mamba install radis
 
+
+**Remark**
+Depending on your platform you should write 
+
+# For Linux:
+pip install radis
```
Comment (erwanp) [inline README.rst:41]: All 3 work in every OS. Let's keep it simple

**Review by erwanp — CHANGES_REQUESTED** [review]

**Comment by minouHub** [comment]
@menasrac please implement the wording change that I suggested + revert the install instructions to the original as suggested by Erwan

**Review by minouHub — APPROVED** [review]

**Pull request merged**